cypress/integration/rewards.spec.js

```diff
@@ -0,0 +1,299 @@
+import Web3 from '@aragon/wrapper/node_modules/web3'
+import { addDays } from 'date-fns'
+
+const PrivateKeyProvider = require('truffle-privatekey-provider')
+const ProviderEngine = require('truffle-privatekey-provider/node_modules/web3-provider-engine')
+const CacheSubprovider = require('truffle-privatekey-provider/node_modules/web3-provider-engine/subproviders/cache.js')
+const FixtureSubprovider = require('truffle-privatekey-provider/node_modules/web3-provider-engine/subproviders/fixture.js')
+const FilterSubprovider = require('truffle-privatekey-provider/node_modules/web3-provider-engine/subproviders/filters.js')
+const VmSubprovider = require('truffle-privatekey-provider/node_modules/web3-provider-engine/subproviders/vm.js')
+const NonceSubprovider = require('truffle-privatekey-provider/node_modules/web3-provider-engine/subproviders/nonce-tracker.js')
+const RpcSubprovider = require('truffle-privatekey-provider/node_modules/web3-provider-engine/subproviders/rpc.js')
+
+var engine = new ProviderEngine()
+
+var web3A = new Web3(engine)
+var web3B = new Web3(engine)
+
+engine.addProvider(
+  new FixtureSubprovider({
+    web3_clientVersion: 'ProviderEngine/v0.0.0/javascript',
+    net_listening: true,
+    eth_hashrate: '0x00',
+    eth_mining: false,
+    eth_syncing: true,
+  })
+)
+
+engine.addProvider(new CacheSubprovider())
+engine.addProvider(new FilterSubprovider())
+engine.addProvider(new NonceSubprovider())
+engine.addProvider(new VmSubprovider())
+
+// data source
+engine.addProvider(
+  new RpcSubprovider({
+    rpcUrl: 'http://localhost:8545',
+  })
+)
+/*
+engine.on('block', function(block) {
```

Comment:
This commented block can be deleted, along with all the other engine-related stuff